RoomLocator/RoomLocator.Domain/Enums/SensorType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace RoomLocator.Domain.Enums
+{
+    public enum SensorType
+    {
+        PeopleCounter = 1,
+        Co2 = 2,
+        Light = 3,
+        Humidity = 4,
+        Ultrasound = 5,
+        Sound = 6,
+        AvailableSeats = 7,
+        MaxAvailableSeats = 8,
+        Temperature = 9
```

Comment:
Det smarteste er ikke at give det tal. Hvis du giver værdierne tal, så bliver det gemt som tal i databasen. Fx vil `Sound` blive gemt som `6` i databasen. Hvis jeg så senere tilføjer en værdi mellem `Ultrasound` og `Sound`, så `Sound` får `7` i stedet for `6`, vil data blive gemt forkert. Det samme gælder i frontenden og med JSON. Tal giver ikke så meget mening.

Man kan få automapperen til skrive navnet på enum i stedet for. Så i stedet for 
```
{
   sensor: {
      "type": 6
   }
}
```
ville man få
```
{
   sensor: {
      "type": "Sound"
   }
}
```